source/connect.txt

```diff
@@ -90,16 +90,32 @@ Steps
       .. image:: /images/connect/intellij-launch-playgrounds.png
          :alt: To launch the Playgrounds, click the highlighted button in Database Explorer.
 
+   .. step:: (*Optional*) Select a data source using the Connections Toolbar.
+
+      The Connections Toolbar is a feature of the {+intellij-full+} that 
+      allows you to select a data source associated with your application
+      from within the editor window.
+      
+      When you open a file that contains MongoDB code, a toolbar appears at 
+      the top of the file as shown in the image below. The Connections 
+      Toolbar is highlighted in red.
+      
+      .. image:: /images/connect/intellij-connections-toolbar.png
+         :alt: The Connections Toolbar appears at the top of the file.
+      
+      This toolbar has a dropdown that allows you to select a data source 
```

Comment:
[style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#:~:text=include%20several%20options.-,drop%2Ddown%20list,-drop%2Ddown%20menu) - we only use drop-down as a hyphenated adjective when referring to a menu or UI element! 
```suggestion
      This toolbar has a drop-down list that allows you to select a data source 
```

Reply:
Thanks for catching this! Done.